### DISCUSSION TITLE
How about write down design document by C# code?

### DISCUSSION THREAD

**Discussion by jongfeel** [Joseph-Cha/JustFight_Client#50] [Show and tell]
중간에 몇 번 얘기해 드리긴 했는데

설계한 문서의 내용대로라면 C# code level로 충분히 표현 가능합니다.

그래서 앞으로는 설계 문서의 내용이 C# code로 나오면 어떨까 해요.

**Comment by Joseph-Cha** [comment]
음 감이 좀 안잡히지만 한번 시도를 해볼게요!
안그래도 설계 문서가 너무 난잡한 느낌이라 어떻게하면 좀더 깔끔하게 만들 수 있을까 고민 하던 차였어요!
